KubeArmor/core/kubeArmor.go

```diff
@@ -448,6 +469,64 @@ func KubeArmor() {
 
 	// == //
 
+	if !dm.K8sEnabled && cfg.GlobalCfg.Policy {
+		// check if the CRI socket set while executing kubearmor exists
+		if cfg.GlobalCfg.CRISocket != "" {
+			trimmedSocket := strings.TrimPrefix(cfg.GlobalCfg.CRISocket, "unix://")
+			if _, err := os.Stat(trimmedSocket); err != nil {
+				dm.Logger.Warnf("Error while looking for CRI socket file: %s", err.Error())
+
+				// destroy the daemon
+				dm.DestroyKubeArmorDaemon()
+				return
+			}
+		}
+
+		if hostContainerRuntime() == "docker" {
```

Comment:
lets keep a local var and then match it
```suggestion
                cri := hostContainerRuntime()
		if cri == "docker" {
```